config/jobs/kubernetes/sig-cloud-provider/azure/release-master.yaml

```diff
@@ -214,6 +214,42 @@ presubmits:
       testgrid-tab-name: pull-kubernetes-e2e-aks-engine-azure-file
       testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
       testgrid-num-columns-recent: '30'
+  - name: pull-kubernetes-e2e-capz-conformance
+    decorate: true
+    always_run: false
+    optional: true
+    decoration_config:
+      timeout: 3h
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+      preset-azure-cred: "true"
+    extra_refs:
+    - org: chewong # TODO(chewong): change it to 'kubernetes-sigs'
+      repo: cluster-api-provider-azure
+      base_ref: upload-build-to-azure # TODO(chewong): change it to 'master'
+      path_alias: sigs.k8s.io/cluster-api-provider-azure
+      workdir: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20210418-e5f251e-master
+        command:
+        - runner.sh
+        - ./scripts/ci-conformance.sh
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            cpu: 1
+            memory: "4Gi"
+        env:
+        - name: KUBETEST_CONF_PATH
+          value: /home/prow/go/src/sigs.k8s.io/cluster-api-provider-azure/test/e2e/data/kubetest/conformance-fast.yaml
```

Comment:
Want to make sure the test duration is less than an hour

Reply:
should we set the timeout to less than 3 hours if that's the goal?

Reply:
removed the timeout (the default is 2h IIRC)